src/containers/DidDocumentView/DidDocumentView.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Contact } from '../../types/Contact'
+import './DidDocumentView.scss'
+import Code from '../../components/Code/Code'
+
+type Props = {
+  didDocument: object
+  children: any
+}
+
+const DidDocumentView = (props: Props) => {
+  const { didDocument } = props
+  return (
+    <>
```

Comment:
Question:
Do you really need to wrap this in a fragment? Why not return `<div className="attributes">` directly?

Reply:
Yes, the fragment does nothing. I have removed it.